pages/builds/parallelizing_builds.md.erb

```diff
@@ -51,29 +51,28 @@ BUILDKITE_PARALLEL_JOB=0
 BUILDKITE_PARALLEL_JOB_COUNT=5
 ```
 
-The `BUILDKITE_PARALLEL_JOB` environment variable is the job number starting from 0, in this case each job’s value would be `0`, `1`, `2`, `3`, and `4`.
+The `BUILDKITE_PARALLEL_JOB` environment variable stores the job number starting from 0. In this case each job’s value would be `0`, `1`, `2`, `3`, and `4` respectively.
 
-The `BUILDKITE_PARALLEL_JOB_COUNT` environment variable is the total number of jobs.
+The `BUILDKITE_PARALLEL_JOB_COUNT` environment variable stores the total number of jobs created from this step for this build.
 
 You can use these two environment variables to divide your application’s tests between the different jobs.
 
 ## Libraries
 
-The following libraries have built-in support for the `BUILDKITE_PARALLEL_*` environment variables:
+The [Knapsack](https://github.com/ArturT/knapsack) ruby gem has built-in support for the `BUILDKITE_PARALLEL_JOB` and `BUILDKITE_PARALLEL_JOB_COUNT` environment variables:
 
-* [Knapsack](https://github.com/ArturT/knapsack)
-<br>A ruby gem for that will automatically divide your tests between parallel jobs, as well as making sure each job runs in comparable time. Supports RSpec, Cucumber and Minitest.
+Knapsack is a ruby gem for automatically dividing your tests between parallel jobs, as well as making sure each job runs in comparable time. It supports RSpec, Cucumber, and Minitest.
 
 ## Isolated jobs
 
 You can safely run multiple agents on a single machine, though you’ll still need to ensure your application’s code is able to be run in parallel. One convenient way of doing this is by using the agent’s [built in Docker and Docker Compose support](docker-containerized-builds) which will run each job in one or more completely isolated containers.
```

Comment:
Never really :) Running multiple agents on the same machine is totes fine under all circumstances!